MandelaK/SendIT-API-V2
# [#162164202] get parcel by id

### What does this PR do?
adds feature allowing users and admins to get parcels by ID

### Description of task to be accomplished
users should be able to get their parcels by ID, while admins should be able to get all parcels by ID. With this feature, admins can get all parcels in the database by their ID, if the parcels exist, while users will only get parcels by ID if they created the requested parcel.

### How can this be manually tested?
Follow the installation instructions on the README page or this repo. From there sign up and log in users and create a few parcels with each one. After that you access the endpoint for getting parcel by ID which is `GET localhost://api/v2/parcels/<int:parcel_id>` ... Use the tokens you were assigned and try to see if each user can get the parcels they created by their ID. Admins should be able to get any parcel by it's ID. Trying to get non-existent parcel should return an error message

### PT stories
#162164202
